wsmaster/codenvy-hosted-api-resource/src/main/java/com/codenvy/resource/spi/impl/FreeResourcesLimitImpl.java

```diff
@@ -0,0 +1,118 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.resource.spi.impl;
+
+import com.codenvy.resource.model.FreeResourcesLimit;
+import com.codenvy.resource.model.Resource;
+
+import org.eclipse.che.account.spi.AccountImpl;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Data object for {@link FreeResourcesLimit}.
+ *
+ * @author Sergii Leschenko
+ */
+@Entity(name = "FreeResourcesLimit")
+@NamedQueries(
+        {
+                @NamedQuery(name = "FreeResourcesLimit.get",
+                            query = "SELECT l FROM FreeResourcesLimit l WHERE l.accountId= :accountId"),
```

Comment:
`SELECT l` may be confused with `SELECT 1`  l, can you please rename it to `lim`, `limit` or something like that.